dvc/repo/init.py

```diff
@@ -82,6 +82,8 @@ def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
         [config.files["repo"], dvcignore, proj.plots.templates.templates_dir]
     )
 
+    logger.info("Initialized DVC repository.\n")
+
     if scm.ignore_file:
         scm.add([os.path.join(dvc_dir, scm.ignore_file)])
         logger.info("\nYou can now commit the changes to git.\n")
```

Comment:
This will make it print two lines between these messages. Let's remove this too.
```suggestion
        logger.info("You can now commit the changes to git.\n")
```